shared/simplestreams/products.go

```diff
@@ -234,7 +241,7 @@ func (s *Products) ToLXD() ([]api.Image, map[string][][]string) {
 					// Locate source image fingerprint
 					var srcFingerprint string
 					for _, item := range srcImage.Items {
-						if item.FileType != "lxd.tar.xz" {
+						if shared.StringInSlice(item.FileType, []string{"lxd.tar.xz", "incus.tar.xz"}) {
```

Comment:
Just checking, but shouldn't here be like this?
```go 
if !shared.StringInSlice(item.FileType, []string{"lxd.tar.xz", "incus.tar.xz"}) { ... }
```

Reply:
hrm maybe, looking...

Reply:
yes good spotting!